.travis.yml

```diff
@@ -39,17 +39,12 @@ jobs:
       arch: amd64
       before_script:
         - sudo apt-get install upx-ucl -y
-        - sudo apt-get install gcc-aarch64-linux-gnu -y
-        - sudo apt-get install libc6-dev-arm64-cross -y
-        - sudo apt-get install gcc-arm-linux-gnueabi -y
-        - sudo apt-get install libc6-dev-armel-cross -y
```

Comment:
why? iirc, this is for crossing build on amd64

Reply:
Now make build in container, no need to install them at host.

Reply:
I mentioned it in the pr descrption.